ProjectTemplates/VisualStudio2013/MonoGameShared/Content/Content.mgcb

```diff
@@ -0,0 +1,15 @@
+
+#----------------------------- Global Properties ----------------------------#
+
+/outputDir:bin/WindowsPhone8
+/intermediateDir:obj/WindowsPhone8
+/platform:WindowsPhone8
```

Comment:
If this is a shared project, is it correct to have `WindowsPhone8` specified in `Content.mgcb`?